src/WebSocketsServiceProvider.php

```diff
@@ -26,23 +27,30 @@ public function boot()
             __DIR__.'/../config/websockets.php' => base_path('config/websockets.php'),
         ], 'config');
 
-        if (! Schema::hasTable('websockets_statistics_entries')) {
-            $this->publishes([
-                __DIR__.'/../database/migrations/create_websockets_statistics_entries_table.php.stub' => database_path('migrations/'.date('Y_m_d_His', time()).'_create_websockets_statistics_entries_table.php'),
-            ], 'migrations');
-        }
+        try {
+            if (! Schema::hasTable('websockets_statistics_entries')) {
```

Comment:
Remove entirely the Schema check, as well as the `use` statement from the top.

Reply:
Removing would create a new migration file on every composer `install` or `update`, as boot command is triggered during package discover

Reply:
I was feeling a bit dumb but from what I know, this is not how `publishes` work: https://laravel.com/docs/7.x/packages#configuration

Reply:
Thanks for reference, this helps understand better with packages.

But this is surprising the `boot` is triggered on `@php artisan package:discover --ansi`

Reply:
The `package:discover` command does that. It needs to register the container and detect the existing Service Providers + the defined service providers in the `config/app.php` file